docs/content/guides/operator/updates.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: Updating a Full Node
```

Comment:
@ebmifa for additional info on release channels, etc

Reply:
@johnjmartin Sorry, do you mean just regular release process info? What channels we use for communication?